packages/alfa-css/src/value/keyword.ts

```diff
@@ -61,7 +61,7 @@ export namespace Keyword {
     return value instanceof Keyword;
   }
 
-  export function parse<T extends string>(...keywords: Array<T>) {
+  export function parse<T extends string>(...keywords: ReadonlyArray<T>) {
```

Comment:
```suggestion
  export function parse<T extends string>(...keywords: Array<T>) {
```

The arrays created for argument spreads are internal to the function and so there's no risk of the caller passing a mutable array that is also mutable by the function:

```js
const f = (...a) => a;
const a = [1, 2, 3];

f(...a) !== a;
f.apply(this, a) !== a;
```